packages/paste-website/src/pages/components/multiselect-combobox/index.mdx

```diff
@@ -206,6 +207,57 @@ it resizes vertically, you can provide a `maxHeight` prop.
   {maxHeightExample}
 </LivePreview>
 
+### useMultiselectCombobox state hook
+
+<Callout variant="warning" marginY="space70">
+  <CalloutHeading as="h5">Power user move!</CalloutHeading>
+  <CalloutText>
+    Only use this property if you are a power user. It's very easy to break your implementation and unfortunately the
+    Paste team will not be able to debug this for you. Proceed with extreme caution.
+  </CalloutText>
+</Callout>
+
+In addition to being a controlled component, the Multiselect Combobox comes with the option
+of "hooking" into the internal state by using the state hook originally provided by
+[Downshift](https://github.com/downshift-js/downshift/tree/master/src/hooks/useCombobox).
```

Comment:
suggestion: link to useMultipleSelection?

Reply:
swapped to the correct link